resolwe_bio/docker_images/rnaseq/packages-manual/trimmomatic.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+
+. /var/cache/build/packages-manual/common.sh
+
+download_and_verify \
+    usadellab \
+    trimmomatic \
+    0.36 \
+    4846c42347b663b9d6d3a8cef30da2aec89fc718bf291392c58e5afcea9f70fe \
+    http://www.usadellab.org/cms/uploads/supplementary/Trimmomatic/Trimmomatic-\${version}.zip \
+    Trimmomatic-\${version}
+
+echo '#!/bin/bash' >>trimmomatic.sh
+echo '' >>trimmomatic.sh
+echo 'java -jar /opt/usadellab/trimmomatic/trimmomatic-0.36.jar "$@"' >>trimmomatic.sh
```

Comment:
Can you do this with only one redirect, e.g. something like
```
cat >trimmomatic.sh <<EOL
#!/bin/bash

java -jar ...
EOL
```

Reply:
Neat, thanks for the suggestion!